department-of-veterans-affairs/caseflow-efolder
# Kev ma/appeals 46542

Resolves [APPEALS-46542](https://jira.devops.va.gov/browse/APPEALS-46542)

### Description
Added ClaimEvidenceApiError to EXCEPTIONS list in order to trigger status failure updates if any errors are caught

### Acceptance Criteria

### Testing Plan
1. Go to UAT console and search up a veteran file number with documents/records
2. Update UAT seed data so that records and manifest source points to an invalid veteran file number
3. Note that :use_ce_api feature toggle is turned on 
4. Note the error banners appear when fetching doucments
